src/common/footer.tsx

```diff
@@ -48,6 +46,11 @@ export function Footer() {
           color: ${textGreen60};
         `}
       >
+        <p>
+          The Tally Ho Wallet is not affiliated with dAppHero and/or the Tally
```

Comment:
This should be “The Tally Ho wallet”, I think, since wallet isn't part of the name.